modules/standard/Memory/Diagnostics.chpl

```diff
@@ -91,6 +91,7 @@
     to its own file, with a dot ('.') and the locale ID appended to
     this path.
  */
+@deprecated("The Memory.Diagnostics module is deprecated - please use :mod:`MemDiagnostics` instead")
```

Comment:
Should we describe ``MemDiagnostics`` as unstable here? It might be useful to give a user that warning up front since unstable warnings are off by default.

But it also doesn't seem necessary. What do you think?

Reply:
I wouldn't have a problem with adding it, something like "please use the unstable :mod:`MemDiagnostics` instead"? But I don't think it is necessary, since we recommend it in several places in the docs and it is used in example code.

Reply:
I think you're right, let's just leave this line as-is.